src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs

```diff
@@ -22,7 +23,7 @@ public static partial class XmlSerializerTests
 
     static XmlSerializerTests()
     {
-        var method = typeof(XmlSerializer).GetMethod(SerializationModeSetterName);
+        var method = typeof(XmlSerializer).GetMethod(SerializationModeSetterName, BindingFlags.NonPublic | BindingFlags.Static);
```

Comment:
Will it be a problem when this test is running against CoreRT if `GetMethod` is not available there?